packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -96,209 +97,1282 @@ export const handler = Sentry.AWSLambda.wrapHandler(
   }
 );
 
-async function getSignedUrl(fileName: string) {
-  return coreGetSignedUrl({ fileName, bucketName, awsRegion: region });
+const jsonToHtml = (json: string): string => {
```

Comment:
would suggest encapsulating all functions related to rendering the html in a separate file, lambda is >1k lines now, and hard to tell what it actually does at a glance